CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 The changes not yet present in any release are listed in this section.
 
+### Added
+
+* Support for ASUS VivoBook 15 X510UA
```

Comment:
Please, make it consistent with other new model items in the changelog (link, dot).